spec/docs/participants_spec.rb

```diff
@@ -146,8 +146,8 @@
     end
   end
 
-  path "/api/v1/participants/{id}/withdraw" do
-    put "Withdraw a participant from a course" do
+  path "/api/v1/participants/{id}/resume" do
+    put "Resume a participant into a course" do
```

Comment:
I'd word this slightly differently to indicate they are "returning to their course after a break". Doesn't read well at the moment.

Reply:
I updated it to have a similar copy to our guidance